test/functional/page_objects/visualize_page.js

```diff
@@ -1012,6 +1012,13 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
       });
     }
 
+    async filterOnTableCell(column, row) {
+      const cell = await find.byCssSelector(`tbody tr:nth-child(${row}) td:nth-child(${column})`);
```

Comment:
nit: what about add a test subject attribute to the table? we can have multiple table on that page